uktrade/enquiry-mgmt-tool
# Add the Non-Applicable enquiry stage

## Description of change

Adds the `Non-applicable` option to _enquiry stage_.

## Test instructions

1. Edit an _enquiry_ of your choice
2. The _Enquiry stage_ field should now have the _Non-applicable_ option
3. Select that option and hit the _Save and return_ button
4. You should now see _Enquiry stage Non-applicable` in the enquiry page header
5. Go to the search page
6. You should now see a new _Non-applicable_ Enquiry stage filter
7. Select only the _Non-applicable_ filter and hit the _Apply filters_ button
8. You should see only the enquiry you edited in the search results, provided you haven't set the _Non-applicable_ stage to more enquiries
